example/streaming-web-demo/server.js

```diff
@@ -0,0 +1,197 @@
+
+    const claudeCode = new ClaudeCode({
+        model: model || undefined,
+        verbose: true,
+        dangerouslySkipPermissions: skipPermissions === 'true',
+        workingDirectory: '/workspace'
+    });
+
+    if (stream === 'true') {
+        console.log(`[${new Date().toISOString()}] Starting streaming request`);
+        
+        // Set up Server-Sent Events
+        res.writeHead(200, {
+            'Content-Type': 'text/event-stream',
+            'Cache-Control': 'no-cache',
+            'Connection': 'keep-alive',
+            'Access-Control-Allow-Origin': '*'
+        });
+
+        try {
+            const response = await claudeCode.chat({
+                prompt: prompt,
+                stream: true
+            });
+
+            console.log(`[${new Date().toISOString()}] Got streaming response`);
+
+            if (response && response.on) {
+                let lastContent = '';
+                
+                // Listen for debug events to capture assistant messages
+                response.on('debug', (event) => {
+                    console.log(`[${new Date().toISOString()}] Debug event:`, JSON.stringify(event));
+                    
+                    // Check if this is an assistant message with content
+                    if (event.type === 'assistant' && event.message && event.message.content) {
+                        const content = event.message.content;
+                        if (Array.isArray(content) && content.length > 0 && content[0].text) {
+                            const fullText = content[0].text;
+                            // Only send the new part
+                            if (fullText.startsWith(lastContent)) {
+                                const newText = fullText.substring(lastContent.length);
+                                if (newText) {
+                                    console.log(`[${new Date().toISOString()}] New text:`, newText);
+                                    res.write(`data: ${JSON.stringify({ type: 'token', content: newText })}\n\n`);
+                                    lastContent = fullText;
+                                }
+                            } else {
+                                // Full new text
+                                console.log(`[${new Date().toISOString()}] Full text:`, fullText);
+                                res.write(`data: ${JSON.stringify({ type: 'token', content: fullText })}\n\n`);
+                                lastContent = fullText;
+                            }
+                        }
+                    }
+                });
+                
+                // Handle standard events
+                response.on('token', (token) => {
+                    console.log(`[${new Date().toISOString()}] Token event:`, token);
+                    res.write(`data: ${JSON.stringify({ type: 'token', content: token })}\n\n`);
+                });

```

Comment:
I tested a few scenarios but didn't manage to make `tokens` emitted. Shall I do any special configurations to receive these events?